3 Longest Substring Without Repeating Characters.js

```diff
@@ -16,7 +16,7 @@ var lengthOfLongestSubstring = function(s) {
 
     // scan from left to right.    
     for(var i = start; i < s.length; i++){
-        c = s[i];
+        var c = s[i];
```

Comment:
maybe let?

Reply:
I agree `let` is better. But to keep consistency I used `var`.

Reply:
👍